services/IoT/applications/libraries/java.md

```diff
@@ -19,19 +19,19 @@ For more information, see [iot-java](https://github.com/ibm-messaging/iot-java)
 ## Constructor
 {: #constructor}
 
-The constructor builds the client instance, and accepts a Properties object that contains the following definitions:
+The constructor builds the client instance and accepts a Properties object that contains the following definitions:
 
 | Definition     |Description     |
 |----------------|----------------|
 |``org`` |Your organization ID. This is a required value. If you are using a Quickstart flow, specify ``quickstart``.|
-|``id``  |The unique ID of the application within your organization.|
-|``auth-method``   |Method of authentication whereby the only value that is currently supported is ``apikey``).|
+|``id``  |The unique ID of the application in your organization.|
+|``auth-method``   |Method of authentication for which the only value that is currently supported is ``apikey``).|
 |``auth-key``   |An API key, which is required when auth-method is set to ``apikey``.|
 |``auth-token``   |An API key token, which is required when auth-method is set to ``apikey``.|
 |``clean-session``|A true or false value that is required only if you want to connect the application in durable subscription mode. By default, ``clean-session`` is set to 'true'.|
 |``shared-subscription``|A boolean value. Set to true if you would like to build scalable applications that balance the load of messages across multiple instances of the application. For more information, see [Scalable applications](https://docs.internetofthings.ibmcloud.com/applications/mqtt.html#/scalable-applications#scalable-applications).
 
-The Properties object creates definitions which are used to interact with the {{site.data.keyword.iot_full}} module. If you do not specify the properties for this object, or if you specify ``quickstart``, the client connects to the {{site.data.keyword.iot_short_notm}} Quickstart service, and defaults to an unregistered device.
+The Properties object creates definitions that are used to interact with the {{site.data.keyword.iot_full}} module. If you do not specify the properties for this object, or if you specify ``quickstart``, the client connects to the {{site.data.keyword.iot_short_notm}} Quickstart service and uses an unregistered device by default.
```

Comment:
do not use default as a verb. I've rewritten to avoid using it as a verb, but someone will have to verify that it is still correct and the meaning hasn't been inadvertently changed.